src/plugins/intel_gpu/src/graph/include/kernel_selector_helper.h

```diff
@@ -129,6 +129,8 @@ struct kernel_impl_params {
 
     memory::ptr reordered_weights = nullptr;
 
+    bool is_dynamic = false;
```

Comment:
Do we need this new flag? Just have a new flag in kernel_selector's base param to have such a flag so that it can be set when the kernel is created?

Reply:
Yes, agree, removed